includes/functions.php

```diff
@@ -2058,39 +2060,104 @@ function tsml_scan_data_source($data_source_url)
 	}
 }
 
-//function:	Returns boolean indicator when data source changes detected
-function tsml_import_has_changes($meetings, $data_source_count_meetings, $data_source_last_refresh)
+//function:	Returns summary list of modified records when data source changes detected
+function tsml_import_has_changes($data_source_url, $meetings, $data_source_count_meetings, $data_source_last_refresh)
```

Comment:
this function name should reflect its new purpose. perhaps `tsml_import_changes`? also `$data_source_count_meetings` is not used and can be removed

Reply:
Good suggestions. Will do....